var/spack/repos/builtin/packages/bolt/package.py

```diff
@@ -40,3 +42,30 @@ def cmake_args(self):
         ]
 
         return options
+
+    @run_after('install')
+    def cache_test_sources(self):
+        """Copy the example source files after the package is installed to an
+        install test subdirectory for use during `spack test run`."""
+        self.cache_extra_test_sources(['examples'])
+
+    def run_sample_nested_example(self):
+        """Run stand alone test: sample_nested"""
+
+        test_dir = join_path(self.install_test_root, 'examples')
```

Comment:
Please use the test stage directory copy for these tests:

```suggestion
        test_dir = join_path(self.test_suite.current_test_cache_dir, 'examples')
```